### ISSUE TITLE
vertical layout of \underset and \overset

### ISSUE THREAD

**Issue by pzinn** [Khan/KaTeX#1288]
Thanks for fixing #1275. This may be nitpicking, but I'm still not quite happy with \underset and \overset.
Compare the output of
```
\overset{a}{b}b\underset{a}{b}b
```
in LaTeX and KaTeX. In LaTeX, all b's are at exactly the same height, whereas in KaTeX the b's wobble.
Is this a known issue/limitation? Only reference to underset/overset I found is #1081 which I think is something different.
I understand that's not top priority, but I thought I'd mention.

**Comment by edemaine** [comment]
KaTeX:
![image](https://user-images.githubusercontent.com/2218736/39663825-ff4f5552-5047-11e8-8f45-2bde7eeeea57.png)

LaTeX (via quicklatex):

![image](https://user-images.githubusercontent.com/2218736/39663828-13c18960-5048-11e8-8be6-851025491f93.png)

The wiggles definitely seem wrong, but so does the horizontal spacing between elements? Ideas, @ronkok?

**Comment by ronkok** [comment]
My first guess is that the base element is being shifted from the baseline to a point where it is centered on the math axis. And my second guess is that we can fix that by changing line 75 in `op.js` from 
```
if (base instanceof domTree.symbolNode) {
```
to 
```
if (group.value.symbol) {
```

In fact, that line actually did read at one time exactly the way I am proposing. Some one had to have changed it and I don't know why. So before I charge in and change it back, I want to think a bit about why they may have done that. Give me a day or two.

**Comment by edemaine** [comment]
@ronkok I traced that particular line back, and it's been there for at least 3 years. Maybe there was another place where you changed it which you're thinking of?

**Comment by ronkok** [comment]
Maybe it was a different line that changed. I still think that line is the most likely way to fix this problem. I'll do some testing.

**Comment by ronkok** [comment]
Fixed by #1290. Thanks for the bug report!
